lte/gateway/python/magma/pipelined/BUILD.bazel

```diff
@@ -9,11 +9,222 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-load("@rules_python//python:defs.bzl", "py_library")
+load("@python_deps//:requirements.bzl", "requirement")
+load("@rules_python//python:defs.bzl", "py_binary", "py_library")
 
 package(default_visibility = ["//visibility:public"])
 
+MAGMA_ROOT = "../../../../../"
+
+ORC8R_ROOT = "{}orc8r/gateway/python".format(MAGMA_ROOT)
+
+LTE_ROOT = "{}lte/gateway/python".format(MAGMA_ROOT)
+
+py_binary(
```

Comment:
The `py_binary` could be private I suppose? But maybe we want to keep this for a larger refactoring of visibilities?

Reply:
You are right, the visibilities are not consistent and should be refactored, but this will be another task.